microcosm_flask/fields/enum_field.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, enum, by_value=False, *args, **kwargs):
         self.by_value = by_value
         super(EnumField, self).__init__(*args, **kwargs)
 
-    def _serialize(self, value, attr, obj):
+    def _serialize(self, value, attr, obj, **kwargs):
```

Comment:
The base class now throws a bunch of extra kwargs in here; subclasses need to respect that signature.